components/dev-tools/scipy/SPECS/python-scipy.spec

```diff
@@ -26,7 +26,7 @@ Requires: openblas-%{compiler_family}%{PROJ_DELIM}
 
 Name:           %{python_prefix}-%{pname}-%{compiler_family}-%{mpi_family}%{PROJ_DELIM}
 Version:        1.5.4
-Release:        1%{?dist}
+Release:        2%{?dist}
```

Comment:
This change is just to trigger the testing for another (non-lmod) module. It should be reverted before merge.